integration_tests/macros/unit_tests/test_get_relation_max_name_length.sql

```diff
@@ -0,0 +1,9 @@
+{% macro test_get_relation_max_name_length() %}
+    {% if target.type == 'redshift'%}
+        {{ assert_value(elementary.get_relation_max_name_length(), 127) }}
```

Comment:
Is it hardcoded in dbt as well? isn't there an adapter.max_name_length or something?